dir.traversal.targets

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="Directory.Build.targets" /> 
+<Project>
```

Comment:
If you're doing this sort of cleanup you could also delete the <?xml ?> tag.

Reply:
which tag?